flagscale/train/hetero/parallel_context.py

```diff
@@ -478,6 +570,8 @@ def __init__(self, args):
 
         self._is_initialized = True
 
+        print(f"keys: {self._global_process_groups.keys()}", flush=True)
```

Comment:
Please remove the original print usage or use a more advanced logging interface.